Completion/Linux/Command/_nmcli

```diff
@@ -1,244 +1,369 @@
-#compdef nmcli
+#compdef _nmcli nmcli
 
-local context state line expl
-typeset -A opt_args
+# compatible with nmcli 1.8.4
 
-if [[ -z $_nmcli_version ]]; then
-    _nmcli_version="${"$(_call_program nmcli $words[1] --version)"##*version }"
-fi
+_nmcli() {
+    local line    _arguments -C \
+        {'(-terse)-t','(-t)-terse'}"[terse output]" \
+        {'(-pretty)-p','(-p)-pretty'}"[pretty output]" \
```

Comment:
Could you perhaps reindent this according the the normal zsh conventions which is just two spaces of indentation. See Etc/completion-style-guide. I understand that the old completion function for nmcli used 4 but as nothing from that has been preserved, there's no need to preserve it to preserve history,